src/test/librbd/managed_lock/test_mock_ReacquireRequest.cc

```diff
@@ -52,7 +52,7 @@ TEST_F(TestMockManagedLockReacquireRequest, Success) {
   C_SaferCond ctx;
   MockReacquireRequest *req = MockReacquireRequest::create(
       mock_image_ctx.md_ctx, mock_image_ctx.header_oid, "old_cookie",
-      "new_cookie", &ctx);
+      "new_cookie", true, &ctx);
```

Comment:
Need test case(s) for shared lock to verify correct setting is getting passed to the cls method